spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -786,18 +786,19 @@ def fixture_record(fixture_name, indexer: @indexer)
 
```

Comment:
It would be a good idea to have the fixture so maybe update the test to include it;
for example on top of the config_spec below line 66 https://github.com/pulibrary/bibdata/blob/601b64baaa0f0b58c3482142b3290059523d5e17/spec/marc_to_solr/lib/config_spec.rb#L66: 
`@siku_9918309193506421 = @indexer.map_record(fixture_record('9918309193506421'))`